package/main/daml/Daml.Finance.Interface.Util/daml.yaml

```diff
@@ -4,7 +4,7 @@
 sdk-version: 2.6.0
 name: daml-finance-interface-util
 source: daml
-version: 1.0.0.99.20230714.1
+version: 1.0.0.99.20230720.1
```

Comment:
Are we actually changing this package? 

It looks like we only touch `D.F.Settlement` and `D.F.Util` (and all dependencies)

Reply:
my nix is currently down, do you mind running `make update-packages validate-packages` and committing the result? it is only daml-finance-util and daml-finance-settlement that changed, and the packages that import those..